src/HTTPClientFactory.ts

```diff
@@ -43,15 +44,18 @@ export class HTTPClientFactory {
 	public static createWithAuthenticationProvider(authProvider: AuthenticationProvider): HTTPClient {
 		const authenticationHandler = new AuthenticationHandler(authProvider);
 		const retryHandler = new RetryHandler(new RetryHandlerOptions());
+		const telemetryHandler = new TelemetryHandler();
 		const httpMessageHandler = new HTTPMessageHandler();
+
```

Comment:
I suggest that you comment about the importance of the order of these handlers and how someone else in the future should update this pipeline of handlers.

Reply:
will be more helpful, will do...